src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -187,6 +197,10 @@ public class ShopifySdk {
 	private String authorizationToken;
 	private WebTarget webTarget;
 	private String accessToken;
+	// everything related this or 'withFulfillmentOrderApi'
+	// is in reference to the deprecation of Shopify's legacy API
+	// starting with 2023-03 API release
+	private boolean useShopifyFulfillmentOrderApi = false;
```

Comment:
Remove this boolean and all the methods associated with it per our conversation with Ryan